_posts/2020-01-30-announcing-kadalu-0.5.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Announcing Kadalu Kubernetes Storage 0.5"
+author: kadalu
+layout: post
+description: "The Kadalu team is happy to announce a new version of Kadalu Kubernetes Storage, 0.5. Kadalu Kubernetes Storage is a lightweight solution to provide storage for applications running in Kubernetes."
+---
+
+The Kadalu team is happy to announce a new version of Kadalu Kubernetes Storage, 0.5. Kadalu Kubernetes Storage is a lightweight solution to provide storage for applications running in Kubernetes.
+
+## New in Kadalu Kubernetes Storage 0.5
+
+### Kadalu Kubectl plugin
+
+Kadalu Kubectl plugin was introduced during the release 0.4, which can be easily installed using `pip3 install kubectl-kadalu`. Kadalu Operator install subcommand is introduced with this release. To install Kadalu on an already running Kubernetes cluster,
```

Comment:
We continue to add capabilities to make it easy to work with the Kadalu Kubectl plugin available since 0.4 release. This release has the install subcommand for the Kadalu Operator thus enabling installation of Kadalu on an already running Kubernetes cluster.